delta/continuedChunk/ChunkedCommand.adoc

```diff
@@ -0,0 +1,27 @@
+[[cmd-ChunkedCommand]]
+==== Chunked Command
+A <<continuedChunk>> for one of the following <<splittableMessage, splittable>> commands:
+
+* <<cmd-AddPartition>> <<AddPartition.newPartition>>
+* <<cmd-AddChild>> <<AddChild.newChild>>
+* <<cmd-ReplaceChild>> <<ReplaceChild.newChild>>
+* <<cmd-AddAnnotation>> <<AddAnnotation.parent>>
+* <<cmd-ReplaceAnnotation>> <<ReplaceAnnotation.newAnnotation>>
+
+.Technical name
+`ChunkedCommand`
+
+[horizontal]
+.Parameters
+[[ChunkedCommand.chunk, `chunk`]]chunk:: <<chunkType>> or <<shallowChunkType>>
```

Comment:
I find it very bad practice to have a property with a join type. Also, I do not see why we need << shallowChunkType>> here. All chunks are full << chunkType >>'s

Reply:
See my comment at ListPartitions, I still see no need for the shallowchunkType there.
Also, I do not think that the number of partitions will ever be big enough to be a problem with the size of this message.

Reply:
Let's say we implement Java AST in LW, and each class becomes a partition. Then every business system easily has tens of thousands of partitions. I think that warrants chunking.